docs/02-app/02-api-reference/05-next-config-js/images.mdx

```diff
@@ -240,3 +241,20 @@ export default function thumborLoader({ src, width, quality }) {
   return `https://example.com${params.join('/')}${src}`
 }
 ```
+
+### Nitrogen AIO
+
+```js
+// Docs: https://docs.n7.io/aio/intergrations/
+export default function aioLoader({ src, width, quality }) {
+  const url = new URL(src)
```

Comment:
```suggestion
  const url = new URL(src, window.location.href)
```
This will fail to handle relative URLs e.g. `new URL('/test.jpg')` will error. Notice the above examples have full URLs.